HeterogeneousCore/CUDAServices/plugins/NVProfilerService.cc

```diff
@@ -831,7 +843,6 @@ void NVProfilerService::postModuleConstruction(edm::ModuleDescription const& des
 void NVProfilerService::preModuleDestruction(edm::ModuleDescription const& desc) {
   if (not skipFirstEvent_) {
     auto mid = desc.id();
-    global_modules_.grow_to_at_least(mid + 1);
```

Comment:
@wddgit this is no longer needed because `global_modules_` will have already been grown in `preModuleConstruction` ?

Reply:
Yes.